public/index.html

```diff
@@ -124,14 +124,21 @@ <h1>MyCargonaut</h1>
                 class="btn btn-sm btn-primary">Alles klar, verstanden!</button></p>
     </div>
 
+    <script src="/__/firebase/7.15.4/firebase-app.js"></script>
+    <script src="/__/firebase/7.15.4/firebase-auth.js"></script>
+    <script src="/__/firebase/7.15.4/firebase-firestore.js"></script>
+    <script src="/__/firebase/7.15.4/firebase-storage.js"></script>
+    <script src="/__/firebase/init.js"></script>
+
     <script src="auth.js"></script>
 
     <script language="JavaScript">
         document.getElementById("cookie-btn").addEventListener("click", function () {
             document.getElementById("cookie").classList.add("d-none");
         });
-        window.firebaseIntegration = new FirebaseIntegration('mycargonaut-b6f0d', 'AIzaSyDekDn6bkOY9I66Nw_ttrA7pX4i7wXATKY',
-                '353710341689', '1:353710341689:web:87b6db09e23c2c7b00d7e8', 'G-V98S2HDFE8');
+        window.firebaseIntegration = new FirebaseIntegration('mycargonaut-b6f0d',
```

Comment:
This line should be removed because init is done by /__/firebase/init.js in Line 121.